code-challenges.test.js

```diff
@@ -16,35 +16,120 @@
 // a) Create a test with an expect statement using the variable provided. 
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+// describe('removeandreShuffle', () => {
+//     const expected = ["purple", "blue"];
+//     it('matches even if received contains additional elements', () => {
+//       expect(["purple", "blue", "green", "yellow", "pink"]).toEqual(expect.arrayContaining(colors1));
+//     });
+//     it('does not match if received does not contain expected elements', () => {
+//       expect(["yellow", "blue", "pink", "green"]).not.toEqual(expect.arrayContaining(colors1));
+//     });
+//   });
+
+// const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+// // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+// const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+// // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+
+// //      Test Suites: 1 passed, 1 total
```

Comment:
The test should not pass before you write the function however your test does look great.  Please leave your code and tests commented in when you submit your work.